### ISSUE TITLE
Game Mechanic - Level One Platforms 

### ISSUE THREAD

**Issue by NimNimR4r** [TeaylThorn/HighwynnGame#6]
**Describe the Bug Part A**
Some players found that their head would go through certain platforms, giving them the impression that they could possibly jump through the platform, when in-fact their head would whip back in a snapping motion... 

***
**Version**
Version One

***
**To Reproduce**
Players would:
1. Jump from certain areas within the level
2. Either hit the platform and have no reaction, they simply couldn't pass through it or
3. Have the head-snapping reaction of not being able to go through the platform 

***
**Expected Behaviour**
Players were expecting to not be able to jump through the platforms. 
When they hit certain areas though, they were mislead to believe that they could jump through, this lead to some confusion as to what they could/could not do... 

***
**Additional Context**
This could be fixed by either having all platforms so that players could not jump through them OR have certain areas (could be a different asset colour) where they can jump up, this could be useful to avoid some enemies 

***
***
**Describe the Bug Part B**
Another issue that some players had, was that they would hit a certain point on some of the ramps that their jump would increase and they would go "flying" up though the level until they either ran out of oomph or they'd hit another platform

***
**Version**
Version One

***
**To Reproduce**
Players would:
1. Run up the ramp
2. Go to jump
3. Hit that spot 
4. Get a super jump and go flying 

***
**Expected Behaviour**
Players were not expecting to get the super jump that they did, especially after the normal jumping movement that they became accustomed to, they weren't ready for it and were a bit stunned when it happened 

***
**Additional Context**
A way to fix this would be to find the cause of the problem and fix it so that our players don't hit that random spot and go flying
(Not sure how we would do this, but it was an issue for some of our play testers)  

***
**TLDR**
* Update platforms to be completely solid so that players cant go through them
* Update certain areas so that players can have the option to jump up through platforms (TBD)
* Update ramps so that players don't hit random spots and go flying (there's one particular ramp that increases the jumping ability of the player)
